app/http.php

```diff
@@ -760,9 +765,11 @@ function (string $runtimeId, string $payload, array $variables, int $timeout, st
             ];
 
             // Update swoole table
-            $runtime = $activeRuntimes->get($activeRuntimeId);
-            $runtime['updated'] = \microtime(true);
-            $activeRuntimes->set($activeRuntimeId, $runtime);
+            $activeRuntimes->set($activeRuntimeId, [
+                'updated' => \microtime(true)
+            ]);
+
+            $activeRuntimes->decr($activeRuntimeId, 'executions', 1);
```

Comment:
It's possible for the count to increment, but never decrement if the code between throws an exception